src/export/InstanceExporter.ts

```diff
@@ -226,6 +226,29 @@ export class InstanceExporter implements Fishable {
             break;
           }
         }
+        // if we don't have instanceChild yet, it may be due to a rule that referes to a named slice somewhere before the end of the path using a numeric index.
```

Comment:
```suggestion
        // if we don't have instanceChild yet, it may be due to a rule that refers to a named slice somewhere before the end of the path using a numeric index.
```